roles/setup_sushy_tools/tasks/main.yml

```diff
@@ -119,3 +119,13 @@
         name: sushy-tools
         state: restarted
         enabled: yes
+      register: sushy_service_result
+
+    - name: Waiting for 1 minute
```

Comment:
I don't understand the purpose of waiting here.

Reply:
I'm not certain about the duration it takes for the sushy-tools service to restart. Do you think it would be more proper to add some sort of retries like this? 

```
- name: Start sushy-tools service
  service:
    name: sushy-tools
    state: restarted
    enabled: yes
  register: sushy_service_result

- name: Wait until sushy-tools service is active
  wait_for:
    timeout: 60
    delay: 5
    state: present
  until: "'active' in sushy_service_result.status.ActiveState"
  retries: 10
```

Reply:
The issue would be (as far as I understand it) that the result doesn't get recomputed so you would have to do something like
```
- name: Start sushy-tools service
  service:
    name: sushy-tools
    state: restarted
    enabled: yes

- name: Check sushy-tools service is active
  service:
    name: sushy-tools
  register: sushy_service_result
  until "'active' in sushy_service_result.status.ActiveState"
  retries: 30
  delay: 5
```

Reply:
thank you for the idea, let me run a job to test it out